### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by tomswartz07**
- Clarify issues found in #364
- Fix a few typos

### CODE REVIEW

File: doc/yay.8 [hunk 1]
```diff
@@ -201,22 +201,22 @@ Sort AUR results by a specific field during search\&.
...
 the same when parsed\&.
 .RE
 .PP
 \fB\-\-answerupgrade\fR <Repo|^Repo|None|...>
 .RS 4
-Set a predetermined answer for the upgrade menu question\&. This answer
-will be used instead of reading from standard input but will be treaded exactly
-the same\&.
+Set a predetermined answer for the upgrade menu question\&. Selects which packages
+groups or repos to select for updades\&. This answer will be used instead of
```
Comment (Morganamilo) [inline doc/yay.8]: This answers what packages to not upgrade. Also you can't enter groups unless you mean `1-3 `which I would call ranges to not confuse with groups like `base` and `base-devel`.

**Comment by Morganamilo** [comment]
There's quite a bit here, thanks for fixing all this. Glad these are all minor typos and there was nothing horribly wrong.

File: doc/yay.8 (hunk 1 above)
Comment (tomswartz07) [reply doc/yay.8]: Ah, apologies- got the logic back-to-front. Updated commit incoming.

**Comment by tomswartz07** [comment]
Glad to help- the program `codespell` helped quite a bit as well :)

File: doc/yay.8 (hunk 1 above)
Comment (Morganamilo) [reply doc/yay.8]: There's still the bit about groups, I would either leave it out or change it to package ranges.

**Comment by Morganamilo** [comment]
I've used aspell in the past, didn't think to check the completion files though. codespell seems really neat, thanks for making me aware of this.

File: doc/yay.8 (hunk 1 above)
Comment (Morganamilo) [reply doc/yay.8]: Looks good now thanks.

**Pull request merged**